docs/details/dashboards/dashboard-sharded-cluster-summary.md

```diff
@@ -0,0 +1,183 @@
+# Sharded Cluster Summary
+
+This dashboard provides a comprehensive view of your MongoDB sharded cluster's performance, health, and resource utilization. It offers insights into shard distribution, query operations, data balance, and replication status across your entire cluster.
+
+![MongoDB Sharded Cluster Summary](../../_images/MongoDB_Sharded_Cluster_Summary.jpg)
+
+## Overview
+
+### QPS of Services in Shard
+ 
+Displays the Queries Per Second (QPS) for each shard and the config server replica set in your MongoDB cluster. It shows the rate of operations (excluding commands) for each component, helping you quickly assess the query load distribution across your sharded environment. 
+
+The chart uses the most recent non-null value and updates based on your selected time interval. This visualization allows you to easily identify which shards are handling the most queries and spot any potential load imbalances. 
+
+### Shards
+
+A shard contains a subset of sharded data for a sharded cluster. Together, the cluster’s shards hold the entire data set for the cluster.
```

Comment:
I think we should add what we are reporting, in this case, the number of shards (as we do below for mongos) rather what a shard is